pkg/api/types.go

```diff
@@ -1345,6 +1345,16 @@ type NodeSpec struct {
 	Unschedulable bool `json:"unschedulable,omitempty"`
 }
 
+// KubeletPorts lists ports opened by Kubelet
+type KubeletPorts struct {
+	// KubeletPort on which Kubelet is listening
+	KubeletPort uint `json:kubeletPort`
+	// KubeletReadOnlyPorty only port open on Kubelet
+	KubeletReadOnlyPorty uint `json:kubeletReadOnlyPort`
```

Comment:
I don't see the ReadOnlyPort used anywhere else in this CL (only the KubeletPort). Nor do I see the CadvisorProxy port used. Why are they being added to the node info?